packages/job-worker/src/playout/adlibAction.ts

```diff
@@ -102,6 +102,7 @@ export async function executeAdlibActionAndSaveModel(
 		userData: data.userData,
 		triggerMode: data.triggerMode,
 		privateData: adLibActionDoc?.privateData,
+		publicData: data.actionOptions,
```

Comment:
I find this unusual. If I were a blueprint developer, when seeing a `publicData` incoming, I would expect that this is `adLibActionDoc?.publicData`, and not something injected by the user/consumer? Also, what's the difference between this implementation of `publicData` and `userData`? Both are incoming from the user/consumer, so why have them split? Why call it `publicData` if it has nothing to do with the entities `publicData`?